doc/building.md

```diff
@@ -107,7 +107,15 @@ of the `legal` directory in the generated image.
 ## Running `jtreg` Self-Tests
 
 The tests can be invoked with individual make targets, or collectively via the
-`test` target.
+`test` target. Individual make targets for self-tests are explained
+[here](../test/README.md#makefiles). For example, the
+[ControlTest.gmk](../test/ctrl/ControlTest.gmk) makefile has a `$(BUILDTESTDIR)/ControlTest.ok`
+target which runs one of the self-tests. In order to run that individual test, use the following
+command:
+
+```shell
+bash build/make.sh -- $(pwd)/build/test/ControlTest.ok
```

Comment:
I don't think you need the `--` when running `build/make.sh`, do you?

Note, you typically _do_ (or may) need `--` when running `make/build.sh`

Reply:
You are correct, removed.